templates/details_table.html

```diff
@@ -5,6 +5,26 @@
 {% block extra_details %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
+      {% if entity.custom_properties.security_classification == "Official-Sensitive"%}
+      <div class="govuk-body govuk-!-padding-top-7 govuk-!-padding-bottom-7">
+        <h2 class="govuk-heading-m">Access this data</h2>
+        <p id="request_access" class="govuk-body"></p>
+          {% if entity.custom_properties.access_information.dc_access_requirements %}
```

Comment:
the indenting of the L13 conditional is a bit off which makes this hard to read - can you make it indented one more level that L12?